server/doc/swdesign/README.md

```diff
@@ -680,6 +680,29 @@ Needs:
 - impl
 - utest
 
+#### Server removes obsolete delete conditions from delete graph
+`swdd~server-removes-obsolete-delete-conditions-from-delete-graph~1`
```

Comment:
As discussed, we should split this requirement in 2 as it is covering a chain of calls.